### PR TITLE
[wasm][debugger] Improve some valuetype visualization

### PR DESCRIPTION
**Pull request by radical**
- Invoke getters for certain types that we know are safe, like `System.DateTime`
- Use `.ToString()` to show as description for some types, like `System.DateTime`
- Show char symbol along with the code

Fixes:
https://github.com/mono/mono/issues/19380
https://github.com/mono/mono/issues/19381
https://github.com/mono/mono/issues/19411

### CODE REVIEW

**Comment by vargaz** [comment]
I think some of these types could be recreated from their components like DateTime:
        private UInt64 dateData;
and ToString () called on the result.

**Comment by radical** [comment]
@monojenkins build Linux WebAssembly

**Comment by lewing** [comment]
@monojenkins build Linux WebAssembly

File: sdks/wasm/src/library_mono.js [hunk 1]
```diff
@@ -562,7 +572,18 @@ var MonoSupportLib = {
...
+		MONO.var_info.push({
+			value: {
+				type: "symbol",
+				value: value,
+				description: value
+			}
+		});
+	},
+
+	mono_wasm_add_obj_var: function(className, toString, objectId) {
```
Comment (lewing) [inline sdks/wasm/src/library_mono.js:586]: let's consolidate several of these similar methods in a follow up by passing the subtype as another argument or as part of the object id

File: sdks/wasm/src/library_mono.js [hunk 2]
```diff
@@ -449,6 +450,15 @@ var MonoSupportLib = {
 		});
 	},
 
+	mono_wasm_add_char_var: function(var_value) {
```
[the remark this answers is not in the document]
Comment (radical) [reply sdks/wasm/src/library_mono.js:453]: I wanted it to be "strongly typed" on JS side, so we could correctly decide what to add as `value: {..}`. I'll clean this up in a future PR, as you suggested in the other comment.

**Comment by lewing** [comment]
@monojenkins build Linux WebAssembly

**Comment by lewing** [comment]
@monojenkins backport 3.2-wasm

**Pull request merged**